### ISSUE TITLE
Package path error in tvm/tests/python/frontend/tflite/test_forward.py

### ISSUE THREAD

**Issue by wangshangsam** [dmlc/tvm#3425]
The package path on line 134:
```
converter = tf.contrib.lite.TFLiteConverter.from_session(
            sess, input_tensors, output_tensors)
```
Should be
```
converter = tf.lite.TFLiteConverter.from_session(
            sess, input_tensors, output_tensors)
```
instead. I'm not sure how TVM's official CI tests manage to pass since then, but it is causing our own version of the CI tests to fail and blocking the testing of our own work. Since the original author has not been responsive, I'll submit a PR to fix it myself.

**Comment by wangshangsam** [comment]
PR #3427

**Comment by wangshangsam** [comment]
@tqchen  @kevinthesun

**Comment by apivovarov** [comment]
TVM supports both tensorflow 1.12 and 1.13
Both allow to use `tf.contrib.lite.TFLiteConverter`.
1.13 will show warning
```
>>> import tensorflow as tf
>>> tf.__version__
'1.13.1'
>>> m = tf.contrib.lite.TFLiteConverter

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

>>> m
<class 'tensorflow.lite.python.lite.TFLiteConverter'>
```
This explains why CI tests work fine.

**Comment by wangshangsam** [comment]
I'm a bit skeptical about this claim. Our CI test builds docker image with tensorflow 1.13 as well, as I just checked (essentially whichever ubuntu_install_tflite.sh specifies).

Regardless, my new commit should fix it for all versions.

**Comment by tqchen** [comment]
closed by #3427
